src/Models/BaseElement.php

```diff
@@ -368,7 +368,7 @@ public function getCMSFields()
      */
     public function getType()
     {
-        return _t(__CLASS__ . '.BlockType', 'Block');
+        return $this->i18n_singular_name();
```

Comment:
```suggestion
        return _t(__CLASS__ . '.BlockType' , $this->i18n_singular_name());
```
Seems like it should be this so that if anyone has defined a specific translation in yml for the block then it should use this before defaulting to i18n_singular_name(), otherwise we'll create a translation regression

Reply:
Yes, this is a great suggestion, thanks @emteknetnz .